commerce_coordinator/apps/core/signal_helpers.py

```diff
@@ -1,10 +1,33 @@
 """
 Commerce Coordinator helper methods for ensuring consistency with Django signal handling.
 """
+import functools
+import inspect
 
 from django.dispatch import Signal
 
 
 class CoordinatorSignal(Signal):
     def send(self, *args, **kwargs):
         raise NotImplementedError("Coordinator Signals do not implement the send method. Use send_robust instead.")
+
+
+def coordinator_receiver(logger):
+    """
+    Return a decorated function with LMS log messages.
+    """
```

Comment:
That's a good idea, to pull in the app name 👍

Reply:
logger.info already has the file path (ex: `INFO 32830 [commerce_coordinator.apps.lms.signals] [user None] [ip None] signal_helpers.py:27` but I can add the part of the message that says "LMS {function} CALLED with..." to be "{app} {function} CALLED with..."

Reply:
Edit: Changing to not have the app name in the message since the module path that is logged already has the app name. 

In the future if we see it really needs to be explicit in uppercase letters for example, we can write a helper function to get the app name (through the name module, Model object, request...)